### DISCUSSION TITLE
Hey

### DISCUSSION THREAD

**Discussion by Bennyskoo** [openenglishbible/Open-English-Bible#350] [General]
Hey I just wanted to say thanks for this project and introduce myself. My name is Chris from Ft. Worth Texas. I had some questions about the NT translation. I emailed the address provided just wondering if you would be available to talk Russell and was curious If you received the email?

**Comment by russellallen** [comment]
Hi Chris - sorry for not responding, have been busy with some personal family stuff. If you send me the email again at oebible(at)openenglishbible.org I promise to reply!  Russell
